ddtrace/monkey.py

```diff
@@ -108,8 +108,14 @@ def _on_import_factory(module, raise_errors=True):
     def on_import(hook):
         # Import and patch module
         path = "ddtrace.contrib.%s" % module
-        imported_module = importlib.import_module(path)
-        imported_module.patch()
+        try:
+            imported_module = importlib.import_module(path)
+        except ImportError:
+            if raise_errors:
+                raise
+            log.error("Failed to import ddtrace module when patching on import", exc_info=True)
```

Comment:
Is there a reason why the module name is not logged here?

Reply:
Nope! Definitely should log it!

I'l make the change in #2139.